app/presenters/order_detail_presenter.rb

```diff
@@ -18,6 +18,15 @@ def description_as_html
     end.join(" &mdash; ").html_safe
   end
 
+  def description_as_text
+    name = product.to_s
+    if bundle
+      name.prepend("#{bundle} -- ")
+    else
+      name
+    end.html_safe
+  end
```

Comment:
this saves duplication but results in some slightly-too-complex code paths, what do you think about flattening some of the logic?